src/api/streams/updateStream.js

```diff
@@ -13,9 +13,13 @@ import { apiPatch } from '../apiFetch';
 export default (
   auth: Auth,
   id: number,
-  property: string,
-  value: string | boolean,
-): Promise<ApiResponse> =>
-  apiPatch(auth, `streams/${id}`, {
-    [property]: value,
-  });
+  params: $ReadOnly<{|
+    description?: string,
+    new_name?: string,
+    is_private?: boolean,
+    is_web_public?: boolean,
+    stream_post_policy?: number,
```

Comment:
```suggestion
    stream_post_policy?: 1 | 2 | 3 | 4,
```

Hmm but actually: I see we don't currently have that property on Stream. Let's add it to Stream and here at the same time; we can just leave it out for now.

In particular that means that before we do add it, we can arrange to draw the connection between this type and Stream, so that we (a) don't have to write this kind of detail twice (it's one thing to do that for like `boolean`, but gets more annoying for something more detailed, and more likely to change in the future, like `1 | 2 | 3 | 4`), and (b) don't risk them getting accidentally out of sync.